SUMMARY.md

```diff
@@ -29,3 +29,10 @@
 - [Glossary](glossary.md)
 - [Links](links.md)
 - [Translations](translations.md)
+
+## Get Involved
+
+- [Contribute](contribute.md)
+- [Community Discord](https://discord.com/invite/olympusdao)
+- [DAO Discord](https://discord.com/invite/TANbneqH)
```

Comment:
Are these permanent invite links? I often see projects linking expired discord links. Just want to make sure we prevent that.

Reply:
```suggestion
- [DAO Discord](https://discord.com/invite/42xFV68uEf)
```

Reply:
i created another DAO discord invitation link with no expiration date